src/config.js

```diff
@@ -0,0 +1,6 @@
+const authenticator = require('./services/authenticator');
+
+module.exports = {
+  authToken: authenticator.getAuthToken(),
+  serverHost: process.env.SERVER_HOST ? process.env.SERVER_HOST : 'https://api.forestadmin.com',
```

Comment:
```suggestion
  serverHost: process.env.SERVER_HOST || 'https://api.forestadmin.com',
```

Reply:
With eslint well configured it would have told you to do that ;)

Reply:
Please, don't keep it for you and update the `.eslintrc` ;)

Reply:
No non I'm just talking about the other configs. I don't have one ;)